kotlin/services/kendra/src/main/kotlin/com/example/kendra/QueryIndex.kt

```diff
@@ -0,0 +1,70 @@
+//snippet-sourcedescription:[QueryIndex.kt demonstrates how to query an Amazon Kendra index.]
+//snippet-keyword:[SDK for Kotlin]
+//snippet-keyword:[Code Sample]
+//snippet-service:[Amazon Kendra]
+//snippet-sourcetype:[full-example]
+//snippet-sourcedate:[03/10/2022]
+//snippet-sourceauthor:[scmacdon - aws]
+
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+package com.example.kendra
+
+// snippet-start:[kendra.kotlin.query.index.import]
+import aws.sdk.kotlin.services.kendra.KendraClient
+import aws.sdk.kotlin.services.kendra.model.QueryRequest
+import aws.sdk.kotlin.services.kendra.model.QueryResultType
+import kotlin.system.exitProcess
+// snippet-end:[kendra.kotlin.query.index.import]
+
+/**
+To run this Kotlin code example, ensure that you have set up your development environment,
+including your credentials.

```

Comment:
Before running this Kotlin code example, set up your development environment,
including your credentials.

Reply:
fixed